05-UI-ReactJS/movies-exercise-redux/src/containers/movieDetails.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
```

Comment:
You have a general concept error right here that is you NEED to use controlled pattern to handle FORMs. The way you are handling this is not fine. Please review it